plugins/modules/instance.py

```diff
@@ -500,6 +514,11 @@
         ],
         editable=True,
     ),
+    "placement_group": SpecField(
+        type=FieldType.dict,
+        suboptions=linode_instance_placement_group_spec,
+        description=["A Placement Group to create this Linode under."],
+    ),
 }
```

Comment:
It looks like re-running a playbook with an instance assigned to a PG and no changes gives the following error:

```
{"changed": false, "msg": "RuntimeError: failed to update PlacementGroup: 1946 -> {'id': 1946, 'compliant_only': False}: placement_group is a non-updatable field"}
```

We may need to introduce some logic in the update function to make sure it doesn't attempt to diff when the PG is unchanged 👀

Reply:
Good catch 👍 I excluded placement_group from the update_params to fix this issue.

Reply:
Thanks for fixing this! Looks like there are some backend issues right now but I'll test the fix as soon as they're resolved